Makefile

```diff
@@ -207,7 +207,7 @@ endif
 # win-sshproxy is checked out manually to keep from pulling in gvisor and it's transitive
 # dependencies. This is only used for the Windows client archives, which must
 # include this lightweight helper binary.
-#
+# Update the SHA in winmake.ps1 as well
 GV_GITURL=https://github.com/containers/gvisor-tap-vsock.git
 GV_SHA=db608827124caa71ba411cec8ea959bb942984fe
```

Comment:
Followup suggestion:  Since (presumably) this is/will-be testing in podman-CI, this would be a really great thing to teach Renovate how to update.  It's regex-manager is more than happy to watch a repo. and propose updated Commit-IDs on a schedule.  Or tags/releases if those are available.  For example, [this bit that keeps golangci-lint updated](https://github.com/containers/automation/blob/6039ae9c9629067e24cb06d5aa53ae1d756f88ad/renovate/defaults.json5#L92-L102) (uses releases though, not SHAs).

Reply:
This particular SHA usage is a bit of a legacy holdover when it was out of sync with the version release stream, but these are now aligned, so it could be a tag. @cfergeau also did just add it as an official release output, so alternatively it could be fetched.

Reply:
I've just made a gvisor-tap-vsock 0.7.1 release with https://github.com/containers/gvisor-tap-vsock/releases/download/v0.7.1/win-sshproxy.exe and https://github.com/containers/gvisor-tap-vsock/releases/download/v0.7.1/gvproxy-windowsgui.exe

Regarding this PR, hardcoding a gvisor-tap-vsock tag/sha1/... in a 3rd place is not very compelling, the macos and windows versions of gvisor-tap-vsock are already out of sync :-/ https://github.com/containers/podman/blob/779c33c336a1a9bb418d6fec34e951a5d14a20e9/contrib/pkginstaller/Makefile#L9

Reply:
Updated to build the latest tag on main, if the ref is not explicitly defined as an arg.

Reply:
Sounds like in this PR or a followup, we should employ Renovate to help keep these all synchronized and updated.